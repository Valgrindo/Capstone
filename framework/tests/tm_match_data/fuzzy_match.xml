<!-- A command that exercises fuzzy matching -->
<commands>

<!--
    Will accept ANY request.
-->
<command name="REQUEST">
    <component indicator="SPEECHACT" type="SA_REQUEST" fuzzy="True" map_param="type"/>
</command>


</commands>