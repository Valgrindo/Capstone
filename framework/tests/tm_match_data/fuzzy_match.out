True; "Give me some money."; {}; {}
True; "Pass me the brush, please."; {}; {}
True; "Go hiking with me"; {}; {}
False; "Can I have some cake?"
False; "My name is John"
False; "Will you marry me?"
False; "I like candy."